### PR TITLE
updated cypress test project

### PR DESCRIPTION
**Pull request by jeevananthank**
# Updated cypress test project

## Description
- removed studio tests
- updated readme and scripts
- updated workflow
- updated code for app-frontend tests

## Fixes
- Altinn/altinn-studio#8049

## Verification
- [x] All tests run green

### CODE REVIEW

File: test/cypress/package.json [hunk 1]
```diff
@@ -1,12 +1,12 @@
 {
-  "name": "cypress-studio",
+  "name": "cypress-altinn-app-frontend",
   "version": "1.0.0",
-  "description": "studio tests",
+  "description": "altinn-app-frontend tests",
   "main": "index.js",
   "config": {
     "localtestPath": "C:/Repos/altinn-studio/src/development",
```
Comment (haakemon) [inline test/cypress/package.json:7]: Whats the significance of these paths? Where are they used?

Comment (jeevananthank) [reply test/cypress/package.json:7]: these paths are used in the scripts [here](https://github.com/Altinn/app-frontend-react/blob/test/remove-studio-tests/test/cypress/package.json#L25), one can change the path as described [here ](https://github.com/Altinn/app-frontend-react/blob/test/remove-studio-tests/test/cypress/package.json#L25) based on the path they have the repo.

**Review by haakemon — APPROVED** [review]
Nice 😁

**Comment by jeevananthank** [comment]
[tests ](https://github.com/Altinn/app-frontend-react/actions/runs/2088186747) have run green.

**Pull request merged**